tracing_tools/perf/cpu_tikv.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Usage: sudo cpu_tikv.sh pid
+
+if [ "$#" -ne 1 ]; then
+  PID=`pgrep tikv-server | head -n 1`
+  echo "Note: PID is not specified. Capturing for tikv-server PID = $PID"
+else
+  PID=$1
+fi
+
+perf record -F 99 -p $PID --call-graph=dwarf sleep 60
```

Comment:
why using dwarf here?

Reply:
It performs notably better when some stack pointers are removed. No "Unknown" any more. Some one said that it also has more overheads. But according to tests, these overheads are small in our TiKV.

Reply:
got it